# HG changeset patch
# User Max Brister <max@2bass.com>
# Date 1346010581 21600
#      Sun Aug 26 13:49:41 2012 -0600
# Node ID d65ef0fc5e05627016215c96843b012b5426ec36
# Parent  44d6ffdf9479e4e6c77c513b59cd84a6a1c474df
configure.ac: Clarify help text on --disable-gui

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -2179,17 +2179,17 @@ AC_SUBST(WARN_CXXFLAGS)
 ### GUI/Qt related tests.
 
 HAVE_QT=false
 QT_INCDIR=
 QT_LIBDIR=
 build_gui=yes
 win32_terminal=no
 AC_ARG_ENABLE(gui,
-  [AS_HELP_STRING([--disable-gui], [build GUI (default is yes)])],
+  [AS_HELP_STRING([--disable-gui], [avoid building the GUI (default no)])],
   [if test "$enableval" = no; then build_gui=no; fi], [build_gui=yes])
 
 if test $build_gui = yes; then
   AC_CHECK_PROGS(QMAKE, [qmake qmake-qt4 qmake-qt5])
   if test -n "$QMAKE"; then
     QT_INCDIR=`$QMAKE -query | sed -n -e 's/^QT_INSTALL_HEADERS://p'`
     QT_LIBDIR=`$QMAKE -query | sed -n -e 's/^QT_INSTALL_LIBS://p'`
     AC_CHECK_PROGS(MOC, [moc moc-qt4 moc-qt5])
